sdk/opa_test.go

```diff
@@ -524,6 +526,122 @@ main = true
 
 }
 
+func TestDecisionWithProvenance(t *testing.T) {
+
+	ctx := context.Background()
+
+	server := sdktest.MustNewServer(
+		sdktest.MockBundle("/bundles/bundle.tar.gz", map[string]string{
+			"main.rego": `
+package system
+
+main = true
+`,
+			".manifest": `{"revision": "v1.0.0"}`,
+		}),
+	)
+
+	defer server.Stop()
+
+	config := fmt.Sprintf(`{
+		"services": {
+			"test": {
+				"url": %q
+			}
+		},
+		"bundles": {
+			"test": {
+				"resource": "/bundles/bundle.tar.gz"
+			}
+		}
+	}`, server.URL())
+
+	opa, err := sdk.New(ctx, sdk.Options{
+		Config: strings.NewReader(config),
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer opa.Stop(ctx)
+
+	result, err := opa.Decision(ctx, sdk.DecisionOptions{})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if decision, ok := result.Result.(bool); !ok || !decision {
+		t.Fatal("expected true but got:", decision, ok)
+	}
+
+	expectedProvenance := types.ProvenanceV1{
+		Version:   version.Version,
```

Comment:
Perhaps we should check that they're not empty strings?

Reply:
I have added a check for this in 51a99515ca445f3c61577b30d219c7584ca07c00